gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -140,103 +109,163 @@ private void addPartitionInfoToBrokerInfo(
   }
 
   @Override
-  public void deployWorkflow(
-      final DeployWorkflowRequest request,
-      final StreamObserver<DeployWorkflowResponse> responseObserver) {
-
-    sendRequest(
-        request,
-        RequestMapper::toDeployWorkflowRequest,
-        ResponseMapper::toDeployWorkflowResponse,
-        responseObserver);
+  public void activateJobs(
+      final ActivateJobsRequest request,
+      final StreamObserver<ActivateJobsResponse> responseObserver) {
+    final BrokerClusterState topology = topologyManager.getTopology();
+    activateJobsHandler.activateJobs(topology.getPartitionsCount(), request, responseObserver);
   }
 
   @Override
-  public void publishMessage(
-      PublishMessageRequest request, StreamObserver<PublishMessageResponse> responseObserver) {
-
+  public void cancelWorkflowInstance(
+      final CancelWorkflowInstanceRequest request,
+      final StreamObserver<CancelWorkflowInstanceResponse> responseObserver) {
     sendRequest(
         request,
-        RequestMapper::toPublishMessageRequest,
-        ResponseMapper::toPublishMessageResponse,
+        RequestMapper::toCancelWorkflowInstanceRequest,
+        ResponseMapper::toCancelWorkflowInstanceResponse,
         responseObserver);
   }
 
   @Override
-  public void updateJobRetries(
-      UpdateJobRetriesRequest request, StreamObserver<UpdateJobRetriesResponse> responseObserver) {
+  public void completeJob(
+      final CompleteJobRequest request,
+      final StreamObserver<CompleteJobResponse> responseObserver) {
     sendRequest(
         request,
-        RequestMapper::toUpdateJobRetriesRequest,
-        ResponseMapper::toUpdateJobRetriesResponse,
+        RequestMapper::toCompleteJobRequest,
+        ResponseMapper::toCompleteJobResponse,
         responseObserver);
   }
 
   @Override
   public void createWorkflowInstance(
-      CreateWorkflowInstanceRequest request,
-      StreamObserver<CreateWorkflowInstanceResponse> responseObserver) {
+      final CreateWorkflowInstanceRequest request,
+      final StreamObserver<CreateWorkflowInstanceResponse> responseObserver) {
+    final BrokerCreateWorkflowInstanceRequest brokerRequest;
+    try {
+      brokerRequest = RequestMapper.toCreateWorkflowInstanceRequest(request);
```

Comment:
What was the previous behaviour here? Did we also return an invalid argument error code? I mean it makes sense, I just want to make sure we're not breaking anything.

Reply:
Previously it was calling `sendRequest` which is doing the same thing.